### PR TITLE
Fix github actions for public gh

### PR DESCRIPTION
**Pull request by miguelalonsojr**
### Proposed change(s)

Fix github actions for public gh

### Types of change(s)

- [X] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [ ] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the changelog (if applicable)
- [ ] Updated the documentation (if applicable)
- [ ] Updated the migration guide (if applicable)

### CODE REVIEW

File: docs/extra.css [hunk 1]
```diff
@@ -1,3 +1,9 @@
 .wy-nav-top, .wy-side-nav-search {
     background: #439b47;
 }
+
+.md-typeset__scrollwrap {
+    margin: 1em -.8rem;
+    overflow-x: visible;
+    touch-action: auto;
+}
```
Comment (maryamhonari) [inline docs/extra.css:9]: some of the changes are coming from the docs change PR, could you rebase and I look again? otherwise looks good

Comment (miguelalonsojr) [reply docs/extra.css:9]: Yep, I did rebase. Not sure why this is coming in as a delta.

Comment (miguelalonsojr) [reply docs/extra.css]: Done.

**Review by maryamhonari — APPROVED** [review]
LGTM assuming tests pass

**Pull request merged**